internal/grpcutil/grpcutil.go

```diff
@@ -40,51 +43,53 @@ func CheckServerVersion(
 		return err
 	}
 
-	version := headerMD.Get(string(responsemeta.ServerVersion))
-	if len(version) == 0 {
-		log.Debug().Msg("error reading server version response header; it may be disabled on the server")
-	} else if len(version) == 1 {
-		currentVersion := version[0]
-
-		// If there is a build on the version, then do not compare.
-		if semver.Build(currentVersion) != "" {
-			log.Debug().Str("this-version", currentVersion).Msg("received build version of SpiceDB")
-			return nil
-		}
+	once.Do(func() {
```

Comment:
why this change?

Reply:
yeah this wasn't obvious: this is run as a middleware on each API call. The `backup restore` command will perform many API calls, so this ended up spamming those messages. It's only really necessary "once per zed session".